airbyte-integrations/connectors/source-facebook-marketing/Dockerfile

```diff
@@ -12,5 +12,5 @@ RUN pip install .
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.2.39
+LABEL io.airbyte.version=0.2.40
```

Comment:
a new connector version is not required since it doesn't actually change the connector's code

Reply:
Updated